dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteRunnable.java

```diff
@@ -852,6 +851,10 @@ public void checkSerialProcess(ProcessDefinition processDefinition) {
             if (nextProcessInstance == null) {
                 return;
             }
+            ProcessInstance nextReadyStopProcessInstance = this.processService.loadNextProcess4Serial(processInstance.getProcessDefinition().getCode(), ExecutionStatus.READY_STOP.getCode(), processInstance.getId());
```

Comment:
Is the state `READY_STOP` right?

Reply:
Yes, this happens when the serial-first workflow is executed at high frequency. If there is an instance in REAY_STOP, it is not possible to wake up the instance that should be executed first.